.devcontainer/Dockerfile

```
@@ -84,6 +86,6 @@ USER devcontainer
 
 # NVM
 RUN curl -o- https://raw.githubusercontent.com/nvm-sh/nvm/v0.39.0/install.sh | bash
-RUN bash --login -i -c 'source /home/devcontainer/.bashrc && nvm install 16'
+RUN bash --login -i -c 'source /home/devcontainer/.bashrc && nvm install 20'
```

```suggestion
RUN bash --login -i -c 'source /home/devcontainer/.bashrc && nvm install 22'
```

22 is the currently used version